### DISCUSSION TITLE
could you update doc/en/xdist.rst?

### DISCUSSION THREAD

**Discussion by cielavenir** [pytest-dev/pytest#8029] [Q&A]
When you google `pytest xdist`, (the first result is pypi xdist and) the second result is https://docs.pytest.org/en/3.0.1/xdist.html , which is managed by https://github.com/pytest-dev/pytest/blob/3.0.1/doc/en/xdist.rst . This file does not exist in latest master (and that is why old version document is getting searched).

The sad point is, it has link http://bitbucket.org/hpk42/execnet/raw/2af991418160/execnet/script/socketserver.py , which does not work anymore. https://github.com/pytest-dev/pytest-xdist has the update link https://raw.githubusercontent.com/pytest-dev/execnet/master/execnet/script/socketserver.py , but I suppose current situation is not so user-friendly.

Could you update doc/en/xdist.rst or have https://docs.pytest.org/en/latest/xdist.html redirect to https://github.com/pytest-dev/pytest-xdist/blob/master/README.rst ?

**Comment by MatejKoplan** [comment]
Agreed. I encountered both of these issues yesterday and I was quite lost when none of the links I found worked. 
Now I see, that the link is fixed on the github page, but not anywhere else. 

Another page that has the wrong info is: 
https://pypi.org/project/pytest-xdist/#description

**Comment by nicoddemus** [comment]
Thanks for the report.

Not sure what to do in this case... should we get the docs for 3.0.1 down? Wouldn't that be a problem for users searching for old docs (although I assume that's extremely rare).
